main.py

```
@@ -3,113 +3,172 @@
+            f"{row.get('base_score','-')} | {row.get('severity','-')} | {row.get('attack_vector','-')} |\n"
+        )
+
+    # CVSS Histogram
+    if not df.empty:
+        df_scores = df.dropna(subset=['base_score'])
+        if not df_scores.empty:
+            fig = px.histogram(df_scores, x="base_score", nbins=10, title="CVSS Score Distribution")
+            try:
+                fig.write_html("docs/cvss_distribution.html")
+                logging.info("CVSS histogram saved to docs/cvss_distribution.html")
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)